backend/tracim_backend/lib/core/reaction.py

```diff
@@ -0,0 +1,84 @@
+import typing
+
+from sqlalchemy.orm import Query
+from sqlalchemy.orm.exc import NoResultFound
+
+from tracim_backend.exceptions import ReactionAlreadyExistError
+from tracim_backend.exceptions import ReactionNotFound
+from tracim_backend.models.auth import User
+from tracim_backend.models.data import Content
+from tracim_backend.models.reaction import Reaction
+from tracim_backend.models.tracim_session import TracimSession
+
+
+class ReactionLib:
```

Comment:
I prefer to avoid Api, i do think this is confusing: https://github.com/tracim/tracim/issues/1103
Api is the legacy one that came from tracimv1 where there was no real "http api".
There is not much logic behind naming choice here, so if you think it's better to use Api, i can change this.

Reply:
Nope, I just wanted to make sure, thanks for the explanation.